client/src/components/Form/Elements/FormCheck.vue

```diff
@@ -0,0 +1,36 @@
+<script setup>
+import { computed, defineProps, defineEmits } from "vue";
+
+const $emit = defineEmits(["input"]);
```

Comment:
I would like to keep the naming of `emit` and `props` consistent across components. I personally favor `emit` over `$emit`, as `$` is an options API prefix to indicate special instance properties.